RealmSwift/SwiftUI.swift

```diff
@@ -643,6 +647,368 @@ extension EnvironmentValues {
             self[RealmEnvironmentKey.self] = newValue.configuration
         }
     }
+    /// The current `PartitionValue` that the view should use.
+    public var partitionValue: PartitionValue? {
+        get {
+            return self[PartitionValueEnvironmentKey.self]
+        }
+        set {
+            self[PartitionValueEnvironmentKey.self] = newValue
+        }
+    }
+}
+
+@available(iOS 14.0, macOS 11.0, tvOS 14.0, watchOS 7.0, *)
+private class ObservableAsyncOpenStorage: ObservableObject {
+    private var app: App
+    var configuration: Realm.Configuration
```

Comment:
These should be private unless needed elsewhere.

Reply:
configuration and partitionValue are overrided when we use them as an environment value, that's why this is not set to private to been able to do that